View/ViewHandler.php

```diff
@@ -448,6 +449,18 @@ private function initResponse(View $view, $format)
             if ($serializer instanceof SerializerInterface) {
                 $context = $this->getSerializationContext($view);
                 $content = $serializer->serialize($data, $format, $context);
+            //       test for symfony 2.7 version 
+            } elseif (class_exists('\Symfony\Component\Serializer\Normalizer\ObjectNormalizer') && $serializer instanceof SymfonySerializerInterface) {
+                    
+                $context = $this->getSerializationContext($view);
```

Comment:
yes .. that being said, maybe some of the tests actually cover too much of how JMS Serializer behaves instead of just ensuring that we pass the right information to the serializer. So it might not be necessary to duplicate all the tests. furthermore, for some tests it will just boil down to a different service injected into the ViewHandler, so we can have a base class for both serialziers.